public/js/components/Charts/OriginCharts.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Themes } from 'formidable-charts';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import AreaChartWrap from './AreaChartWrap';
+import ScatterChartWrap from './ScatterChartWrap';
+
+const OriginCharts = ({ charts, isUpdating, toggleStackChart, filterVals }) => {
+
+    const renderTitle = (type) => {
+        switch (type) {
+        case 'originatingSystem':
+            return <h3>Tool of Origin: <span className='chart-title--second-series'>InCopy</span> vs <span className='chart-title--first-series'>Composer</span></h3>;
+        case 'workflowState':
+            return <h3>Workflow State: <span className='chart-title--second-series'>Never Seen in Workflow</span> vs <span className='chart-title--first-series'>Seen in Workflow</span></h3>;
+        default:
+            '';
+        }
+    };
+
+    const getClassName = (error) => {
+        if (error) {
+            return 'chart-wrap chart-wrap__error';
+        } else {
+            return isUpdating ? 'chart-wrap chart-wrap__updating' : 'chart-wrap';
+        }
+    };
+
```

Comment:
Do you want to pull these outside of the parent function as they are pure and don't need to be recomputed on every render.